tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_MagnetizedTovStar.cpp

```diff
@@ -53,18 +52,19 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticData.GrMhd.MagTovStar",
   std::unique_ptr<EquationsOfState::EquationOfState<true, 1>> eos =
       std::make_unique<EquationsOfState::PolytropicFluid<true>>(100.0, 2.0);
 
-  const gr::Solutions::TovSolution tov_soln{*eos, 1.28e-3};
-  const Mesh<3> mesh{{{5, 5, 5}},
-                     {{Spectral::Basis::Legendre, Spectral::Basis::Legendre,
-                       Spectral::Basis::Legendre}},
-                     {{Spectral::Quadrature::Gauss, Spectral::Quadrature::Gauss,
-                       Spectral::Quadrature::Gauss}}};
+  const Mesh<3> mesh{
```

Comment:
Why change this? It makes it less obvious that the huge refactor you did doesn't change numerical values

Reply:
I think I did this so the test includes a point at the origin. I'll find another way

Reply:
hmm, but 5x5x5 Gauss and GL will both have a grid point at the origin

Reply:
Below a `1.` is added to the logical coords, so currently I think the origin is not included with Gauss points. Easy enough to add tho

Reply:
ah okay, I understand. Okay, I'm fine with you leaving as-is :)